tasks/install.yml

```diff
@@ -65,4 +54,10 @@
   args:
```

Comment:
use `become_user` for those tasks

Reply:
no because passing the enviroment does not work all the time using become